experiment/find_developers.py

```diff
@@ -24,6 +24,7 @@
 import os
 import random
 import sys
+from functools import total_ordering
```

Comment:
Please import modules not methods

http://google.github.io/styleguide/pyguide.html#22-imports

Reply:
👍 I will fix this. Also, I will delete this PR and reintroduce the changes *by module* (e.g. `experiment/module`).